packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -414,7 +421,23 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
     private process(message: ISequencedDocumentMessage, local: boolean, localOpMetadata: unknown) {
         this.verifyNotClosed(); // This will result in container closure.
         this.emit("pre-op", message, local, this);
-        this.processCore(message, local, localOpMetadata);
+
+        // Do not measure for local ops, because callbacks are not invoked for them in this codepath
+        // and including them in the measurements would skew the numbers.
+        if (local) {
+            this.processCore(message, local, localOpMetadata);
+        } else {
+            ExecuteWithAggregatedTelemetry(
+                SharedObjectCore.DdsOpProcessingTelemetrySymbol,
+                () => { this.processCore(message, local, localOpMetadata); },
+                100,
```

Comment:
100 is probably a fine default, but we should make this configuration driven, as tuning via code will be costly and slow.